lib/interface/collection/artist.dart

```diff
@@ -180,6 +180,9 @@ class _ArtistTabState extends State<ArtistTab> {
                             top: MediaQuery.of(context).padding.top +
                                 kMobileSearchBarHeight +
                                 2 * tileMargin,
+                            bottom: Configuration.instance.stickyMiniplayer
```

Comment:
Other screens e.g. `AlbumTab`, `TrackTab` do not contain this bottom padding (except this `ArtistTab`). Even if `Configuration.instance.stickyMiniplayer` becomes `false`, `kMobileNowPlayingBarHeight` is added, which wasn't here before.

Reply:
most screens are nested inside CustomListView which has a padding for miniplayer, some screens can be nested inside `CustomListView` and others can't, thats why manual padding is added, as for `AlbumTab` i remember that i did add a padding? sorry for this.

As for the second part
```
bottom: Configuration.instance.stickyMiniplayer
                    ? kMobileNowPlayingBarHeight
                    : kMobileBottomPaddingSmall
```
wouldn't ~`kMobileBottomPaddingSmall`~ ` kMobileBottomPaddingStickyMiniplayer` be added if `stickyMiniplayer` becomes false?